### PR TITLE
viewApidoc3

### PR DESCRIPTION
**Pull request by djflexible**
View apidoc on our server.

### CODE REVIEW

File: src/server/routes/users.route.js [hunk 1]
```diff
@@ -147,6 +147,7 @@ router.post('/secretCode', user.requestSecretCode);
  *     HTTP/1.1 400
  *     {
  *       "err_msg": "This secret code had expired.."
+ *       "err_msg": "This secret code had expired.."
```
Comment (SarahhhC) [inline src/server/routes/users.route.js]: Please check this one, duplicated!

File: src/server/routes/users.route.js [hunk 2]
```diff
@@ -156,6 +157,7 @@ router.post('/secretCode', user.requestSecretCode);
  *     }
  *
  * @apiDescription If the secret code is valid, the password will be updated.
+>>>>>>> 1916cd1c1fa24787f9519ceb85d71192cbe72ee1
```
Comment (SarahhhC) [inline src/server/routes/users.route.js]: Remove this line.

Comment (djflexible) [inline src/server/routes/users.route.js]: Oh I see!

**Comment by SarahhhC** [comment]
Good! Thanks @djflexible 👍 LGTM

**Pull request merged**